lib/cw-core/src/tile.rs

```diff
@@ -207,12 +208,12 @@ impl Tile {
 
         // Generate framebuffer from pointer and generating the CRC by taking the sum of all the RGB values of the framebuffer
         let mut frame_buffer = [0; 1201];
-        let mut crc: u8 = 0;
+        let mut crc: usize = 0;
         for (i, byte) in frame_buffer_unordered.into_iter().enumerate() {
-            crc += *byte;
+            crc += *byte as usize;
             frame_buffer[self.index_converter_vector[i]] = *byte;
         }
-        frame_buffer[1200] = crc;
+        frame_buffer[1200] = (crc % 256 )as u8;
```

Comment:
the CRC is intialised as usize so that the maximum sum will be the max of usize. Otherwise, when the CRC sum gets larger than the max of u8, the system would be cracked due to oversize is not allowed in Rust.

Reply:
You can keep your implementation, because I do find it cleaner now then when I was implementing this a couple months ago. 

However, the overflow error should not occur since in the `Cargo.toml` file I have put this line [`overflow-checks = false`](https://github.com/StrijpT-Ellie/contour-wall/blob/main/lib/cw-core/Cargo.toml), which should turn of the runtime overflow checking. If you want to know more about this you should read this paragraph of the [Rust book](https://doc.rust-lang.org/book/ch03-02-data-types.html#integer-overflow), here is also the [documentation ](https://doc.rust-lang.org/cargo/reference/profiles.html#overflow-checks)on this profile for Cargo.